core/src/main/resources/META-INF/mule-module.properties

```diff
@@ -41,6 +41,7 @@ artifact.export.classPackages=org.mule.runtime.core,\
                               org.mule.runtime.core.api.routing.filter,\
                               org.mule.runtime.core.api.rx,\
                               org.mule.runtime.core.api.scheduler,\
+                              org.mule.runtime.core.api.scheduler.exception,\
```

Comment:
It will get to the source, so it will need to be able to load the class for the exception on whatever classloader the source is